index.html

```diff
@@ -5079,6 +5079,8 @@ <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
     <section class="appendix informative">
       <h2>IANA Considerations</h2>
+<section id="json-web-token-claim-registrations">
+  <h2>JSON Web Token Claim Registrations</h2>
```

Comment:
To match the other two retitled sections...
```suggestion
  <h2>JSON Web Token Claim Registrations: `vc` and `vp`</h2>
```